docs/src/api.md

```diff
@@ -109,10 +109,10 @@ OSM.latlon
 OSM.intersection
 OSM.road
 OSM.random_road_position
-OSM.plan_route
+OSM.plan_route!
```

Comment:
I'm just a bit confused here. I thought `plan_route!` and `move_along_route!` would become top-level API functions. Why are they still in the OSM module then?